WcaOnRails/db/migrate/20170916165728_change_incident_status.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class ChangeIncidentStatus < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
@viroulep, it looks like you changed your mind about the schema while you were working on this project? It would be nicer for your code reviewers if you fix the old migrations rather than adding new ones =)

Reply:
Right, I'll keep this in mind for my next "big" change!